ci-operator/config/stolostron/clc-ui-e2e/stolostron-clc-ui-e2e-main.yaml

```diff
@@ -0,0 +1,25 @@
+base_images:
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+images:
+- dockerfile_path: Dockerfile.interops
+  to: clc-ui-e2e
+promotion:
+  name: "1.0"
+  namespace: acm-qe
+resources:
```

Comment:
Just curious, is the resources block required if you are just promoting an image?

Reply:
Just checked and it still is required. This tells me that this value must be used for the actually src-build or custom build pods.

Reply:
Sounds good, thanks for clarifying.